projects/packages/publicize/src/class-publicize.php

```diff
@@ -431,7 +429,7 @@ public function refresh_connections() {
 		if ( get_transient( self::CONNECTION_REFRESH_WAIT_TRANSIENT ) ) {
 			return;
 		}
-		$xml = new Jetpack_IXR_Client();
+		$xml = new \Jetpack_IXR_Client();
```

Comment:
`Jetpack_Options` and `Jetpack_IXR_Client` will need to be added as dependencies to the Publicize package as well.